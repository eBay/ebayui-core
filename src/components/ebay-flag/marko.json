{
    "taglib-id": "ebayui-core-flags",
    "tags-dir": "./icons"
}
