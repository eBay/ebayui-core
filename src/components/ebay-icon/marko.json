{
  "taglib-id": "ebayui-core-icons",
  "tags-dir": "./icons"
}