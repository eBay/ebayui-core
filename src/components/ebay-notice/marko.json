{
    "<ebay-notice>": {
        "renderer": "./index.js"
    }
}
