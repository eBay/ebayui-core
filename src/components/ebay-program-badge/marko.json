{
  "taglib-id": "ebayui-core-program-badges",
  "tags-dir": "./icons"
}
