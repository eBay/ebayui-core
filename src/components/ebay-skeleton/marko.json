{
    "taglib-id": "ebayui-core-skeleton",
    "tags-dir": "./components"
}
