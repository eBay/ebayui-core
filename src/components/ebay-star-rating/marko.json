{
    "taglib-id": "ebayui-core-star-rating",
    "tags-dir": "./icons"
}
