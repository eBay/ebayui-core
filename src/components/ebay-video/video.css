.video-player {
    position: relative;
}

.video-player video {
    height: 100%;
    width: 100%;
}

.video-player button {
    font-family: inherit;
}

.video-player__overlay {
    align-items: center;
    background-color: rgba(0, 0, 0, 0.6);
    bottom: 0;
    color: var(--color-neutral-100);
    cursor: pointer;
    display: flex;
    flex-direction: column;
    justify-content: center;
    left: 0;
    position: absolute;
    right: 0;
    text-align: center;
    top: 0;
}

.video-player__overlay-text {
    margin: 20px 20px 0;
}

.video-player__overlay--hidden {
    display: none;
}

.video-player__report-button > svg {
    margin-inline-end: var(--spacing-50);
}

.video-player .shaka-overflow-menu svg {
    height: 24px;
    margin-left: 10px;
    margin-right: 10px;
    width: 24px;
}

.video-player .shaka-play-button {
    background: none;
    box-shadow: none;
    height: auto;
    padding: 0;
    width: auto;
}

.video-player .icon--video-play,
.video-player .shaka-play-button svg {
    height: 48px;
    width: 48px;
}

.video-player .shaka-bottom-controls {
    box-sizing: border-box;
    width: 94%;
    padding-bottom: 0;
    margin-bottom: 3%;
    position: relative;
}

.video-player .shaka-controls-button-panel {
    background-color: rgba(0, 0, 0, 0.7);
    border-radius: 8px;
    position: absolute;
    width: 100%;
    bottom: 0;
    height: 40px;

    :first-child {
        margin-left: 8px;
    }
    :last-child {
        margin-right: 8px;
    }
}

.video-player--poster .shaka-scrim-container {
    visibility: hidden;
}

.video-player .shaka-range-container {
    box-sizing: border-box;
    margin: 0 168px 18px 76px;
}

.video-player #shaka-player-ui-time-container {
    display: none;
}

.video-player .shaka-video-container {
    font-family: "Market Sans", Arial, sans-serif;
}
