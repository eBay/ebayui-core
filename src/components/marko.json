{
  "<ebay-tooltip-base>": {
    "renderer": "./components/ebay-tooltip-base/index.js"
  },
  "<ebay-tooltip-overlay>": {
    "renderer": "./components/ebay-tooltip-overlay/index.js"
  },
  "<*>": {
    "transformer": "../common/transformers/ebayui-attribute/index.js"
  }
}