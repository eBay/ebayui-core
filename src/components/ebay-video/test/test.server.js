import { describe, it } from "vitest";
import { composeStories } from "@storybook/marko";
import { snapshotHTML } from "../../../common/test-utils/snapshots";
import * as stories from "../video.stories";

const { Default, ios, mp4 } = composeStories(stories);

const htmlSnap = snapshotHTML(__dirname);

describe("video", () => {
    it("renders default", async () => {
        await htmlSnap(Default);
    });
    it("renders ios video player", async () => {
        await htmlSnap(ios);
    });
    it("renders with mp4", async () => {
        await htmlSnap(mp4);
    });
    it("renders with report text", async () => {
        await htmlSnap(Default, { reportText: "This is a report text" });
    });
});
