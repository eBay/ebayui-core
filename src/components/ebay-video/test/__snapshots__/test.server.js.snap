// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`video > renders default 1`] = `
"[36m<DocumentFragment>[39m
  [36m<div[39m
    [33mclass[39m=[32m"video-player video-player--poster"[39m
  [36m>[39m
    [36m<div[39m
      [33mclass[39m=[32m"video-player__container"[39m
      [33mdata-marko-key[39m=[32m"1 s0"[39m
      [33mstyle[39m=[32m"width:600px;height:400px"[39m
    [36m>[39m
      [36m<video[39m
        [33mno-controls[39m=[32m""[39m
      [36m>[39m
        [36m<source[39m
          [33msrc[39m=[32m"https://ir.ebaystatic.com/cr/v/c1/ebayui/video/v1/playlist.mpd"[39m
          [33mtype[39m=[32m"dash"[39m
        [36m/>[39m
      [36m</video>[39m
    [36m</div>[39m
    [36m<div[39m
      [33mclass[39m=[32m"video-player__overlay video-player__overlay--hidden"[39m
    [36m>[39m
      [36m<svg[39m
        [33maria-hidden[39m=[32m"true"[39m
        [33mclass[39m=[32m"icon icon--24"[39m
        [33mfocusable[39m=[32m"false"[39m
      [36m>[39m
        [36m<defs>[39m
          [36m<symbol[39m
            [33mid[39m=[32m"icon-attention-24"[39m
            [33mviewBox[39m=[32m"0 0 24 24"[39m
          [36m>[39m
            [36m<path[39m
              [33md[39m=[32m"M12 0C5.373 0 0 5.373 0 12s5.373 12 12 12 12-5.373 12-12C23.994 5.375 18.625.006 12 0Zm0 22C6.477 22 2 17.523 2 12S6.477 2 12 2s10 4.477 10 10c-.006 5.52-4.48 9.994-10 10ZM11 7a1 1 0 1 1 2 0v6a1 1 0 1 1-2 0V7Zm1 11a1 1 0 1 0 0-2 1 1 0 0 0 0 2Z"[39m
            [36m/>[39m
          [36m</symbol>[39m
        [36m</defs>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-attention-24"[39m
        [36m/>[39m
      [36m</svg>[39m
      [36m<div[39m
        [33mclass[39m=[32m"video-player__overlay-text"[39m
      [36m>[39m
        [0mAn error has occurred[0m
      [36m</div>[39m
    [36m</div>[39m
    [36m<div[39m
      [33mclass[39m=[32m"video-player__overlay video-player__overlay--hidden"[39m
    [36m>[39m
      [36m<svg[39m
        [33maria-labelledby[39m=[32m"s0-9-0-text"[39m
        [33mclass[39m=[32m"icon icon--64-colored"[39m
        [33mfocusable[39m=[32m"false"[39m
        [33mrole[39m=[32m"img"[39m
      [36m>[39m
        [36m<defs>[39m
          [36m<symbol[39m
            [33mid[39m=[32m"icon-play-filled-64-colored"[39m
            [33mviewBox[39m=[32m"0 0 64 64"[39m
          [36m>[39m
            [36m<circle[39m
              [33mcx[39m=[32m"32"[39m
              [33mcy[39m=[32m"32"[39m
              [33mfill[39m=[32m"#fff"[39m
              [33mfill-opacity[39m=[32m".88"[39m
              [33mr[39m=[32m"32"[39m
            [36m/>[39m
            [36m<path[39m
              [33md[39m=[32m"m26.131 16.41 21.6 13.33a2.648 2.648 0 0 1 0 4.52l-21.6 13.33C24.333 48.7 22 47.425 22 45.33V18.67c0-2.094 2.333-3.37 4.131-2.26Z"[39m
              [33mfill[39m=[32m"#191919"[39m
            [36m/>[39m
          [36m</symbol>[39m
        [36m</defs>[39m
        [36m<title[39m
          [33mid[39m=[32m"s0-9-0-text"[39m
        [36m>[39m
          [0mClick to play[0m
        [36m</title>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-play-filled-64-colored"[39m
        [36m/>[39m
      [36m</svg>[39m
    [36m</div>[39m
    [36m<div[39m
      [33mstyle[39m=[32m"display:none"[39m
    [36m>[39m
      [36m<svg[39m
        [33maria-hidden[39m=[32m"true"[39m
        [33mclass[39m=[32m"icon icon--64-colored"[39m
        [33mfocusable[39m=[32m"false"[39m
      [36m>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-play-filled-64-colored"[39m
        [36m/>[39m
      [36m</svg>[39m
      [36m<svg[39m
        [33maria-hidden[39m=[32m"true"[39m
        [33mclass[39m=[32m"icon icon--24"[39m
        [33mfocusable[39m=[32m"false"[39m
      [36m>[39m
        [36m<defs>[39m
          [36m<symbol[39m
            [33mid[39m=[32m"icon-flag-24"[39m
            [33mviewBox[39m=[32m"0 0 24 24"[39m
          [36m>[39m
            [36m<path[39m
              [33mclip-rule[39m=[32m"evenodd"[39m
              [33md[39m=[32m"M20.777 1.37a.999.999 0 0 1-.207 1.452L14.72 7l5.832 4.166A.999.999 0 0 1 19.979 13H7v9a1 1 0 1 1-2 0V2a1 1 0 0 1 1-1h13.987a1.001 1.001 0 0 1 .79.37ZM16.88 11l-4.46-3.186a1 1 0 0 1 0-1.628L16.879 3H7v8h9.879Z"[39m
              [33mfill-rule[39m=[32m"evenodd"[39m
            [36m/>[39m
          [36m</symbol>[39m
        [36m</defs>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-flag-24"[39m
        [36m/>[39m
      [36m</svg>[39m
    [36m</div>[39m
  [36m</div>[39m
[36m</DocumentFragment>[39m"
`;

exports[`video > renders ios video player 1`] = `
"[36m<DocumentFragment>[39m
  [36m<div[39m
    [33mclass[39m=[32m"video-player video-player--poster"[39m
  [36m>[39m
    [36m<div[39m
      [33mclass[39m=[32m"video-player__container"[39m
      [33mdata-marko-key[39m=[32m"1 s0"[39m
      [33mstyle[39m=[32m"width:600px;height:400px"[39m
    [36m>[39m
      [36m<video[39m
        [33mno-controls[39m=[32m""[39m
        [33mshaka-config[39m=[32m"{\\"streaming\\":{\\"lowLatencyMode\\":true}}"[39m
      [36m>[39m
        [36m<source[39m
          [33msrc[39m=[32m"https://ir.ebaystatic.com/cr/v/c1/ebayui/video/v1/playlist.m3u8"[39m
          [33mtype[39m=[32m"hls"[39m
        [36m/>[39m
        [36m<source[39m
          [33msrc[39m=[32m"https://ir.ebaystatic.com/cr/v/c1/ebayui/video/v1/playlist.mpd"[39m
          [33mtype[39m=[32m"dash"[39m
        [36m/>[39m
      [36m</video>[39m
    [36m</div>[39m
    [36m<div[39m
      [33mclass[39m=[32m"video-player__overlay video-player__overlay--hidden"[39m
    [36m>[39m
      [36m<svg[39m
        [33maria-hidden[39m=[32m"true"[39m
        [33mclass[39m=[32m"icon icon--24"[39m
        [33mfocusable[39m=[32m"false"[39m
      [36m>[39m
        [36m<defs>[39m
          [36m<symbol[39m
            [33mid[39m=[32m"icon-attention-24"[39m
            [33mviewBox[39m=[32m"0 0 24 24"[39m
          [36m>[39m
            [36m<path[39m
              [33md[39m=[32m"M12 0C5.373 0 0 5.373 0 12s5.373 12 12 12 12-5.373 12-12C23.994 5.375 18.625.006 12 0Zm0 22C6.477 22 2 17.523 2 12S6.477 2 12 2s10 4.477 10 10c-.006 5.52-4.48 9.994-10 10ZM11 7a1 1 0 1 1 2 0v6a1 1 0 1 1-2 0V7Zm1 11a1 1 0 1 0 0-2 1 1 0 0 0 0 2Z"[39m
            [36m/>[39m
          [36m</symbol>[39m
        [36m</defs>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-attention-24"[39m
        [36m/>[39m
      [36m</svg>[39m
      [36m<div[39m
        [33mclass[39m=[32m"video-player__overlay-text"[39m
      [36m>[39m
        [0mAn error has occurred[0m
      [36m</div>[39m
    [36m</div>[39m
    [36m<div[39m
      [33mclass[39m=[32m"video-player__overlay video-player__overlay--hidden"[39m
    [36m>[39m
      [36m<svg[39m
        [33maria-labelledby[39m=[32m"s0-9-0-text"[39m
        [33mclass[39m=[32m"icon icon--64-colored"[39m
        [33mfocusable[39m=[32m"false"[39m
        [33mrole[39m=[32m"img"[39m
      [36m>[39m
        [36m<defs>[39m
          [36m<symbol[39m
            [33mid[39m=[32m"icon-play-filled-64-colored"[39m
            [33mviewBox[39m=[32m"0 0 64 64"[39m
          [36m>[39m
            [36m<circle[39m
              [33mcx[39m=[32m"32"[39m
              [33mcy[39m=[32m"32"[39m
              [33mfill[39m=[32m"#fff"[39m
              [33mfill-opacity[39m=[32m".88"[39m
              [33mr[39m=[32m"32"[39m
            [36m/>[39m
            [36m<path[39m
              [33md[39m=[32m"m26.131 16.41 21.6 13.33a2.648 2.648 0 0 1 0 4.52l-21.6 13.33C24.333 48.7 22 47.425 22 45.33V18.67c0-2.094 2.333-3.37 4.131-2.26Z"[39m
              [33mfill[39m=[32m"#191919"[39m
            [36m/>[39m
          [36m</symbol>[39m
        [36m</defs>[39m
        [36m<title[39m
          [33mid[39m=[32m"s0-9-0-text"[39m
        [36m>[39m
          [0mClick to play[0m
        [36m</title>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-play-filled-64-colored"[39m
        [36m/>[39m
      [36m</svg>[39m
    [36m</div>[39m
    [36m<div[39m
      [33mstyle[39m=[32m"display:none"[39m
    [36m>[39m
      [36m<svg[39m
        [33maria-hidden[39m=[32m"true"[39m
        [33mclass[39m=[32m"icon icon--64-colored"[39m
        [33mfocusable[39m=[32m"false"[39m
      [36m>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-play-filled-64-colored"[39m
        [36m/>[39m
      [36m</svg>[39m
      [36m<svg[39m
        [33maria-hidden[39m=[32m"true"[39m
        [33mclass[39m=[32m"icon icon--24"[39m
        [33mfocusable[39m=[32m"false"[39m
      [36m>[39m
        [36m<defs>[39m
          [36m<symbol[39m
            [33mid[39m=[32m"icon-flag-24"[39m
            [33mviewBox[39m=[32m"0 0 24 24"[39m
          [36m>[39m
            [36m<path[39m
              [33mclip-rule[39m=[32m"evenodd"[39m
              [33md[39m=[32m"M20.777 1.37a.999.999 0 0 1-.207 1.452L14.72 7l5.832 4.166A.999.999 0 0 1 19.979 13H7v9a1 1 0 1 1-2 0V2a1 1 0 0 1 1-1h13.987a1.001 1.001 0 0 1 .79.37ZM16.88 11l-4.46-3.186a1 1 0 0 1 0-1.628L16.879 3H7v8h9.879Z"[39m
              [33mfill-rule[39m=[32m"evenodd"[39m
            [36m/>[39m
          [36m</symbol>[39m
        [36m</defs>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-flag-24"[39m
        [36m/>[39m
      [36m</svg>[39m
    [36m</div>[39m
  [36m</div>[39m
[36m</DocumentFragment>[39m"
`;

exports[`video > renders with mp4 1`] = `
"[36m<DocumentFragment>[39m
  [36m<div[39m
    [33mclass[39m=[32m"video-player video-player--poster"[39m
  [36m>[39m
    [36m<div[39m
      [33mclass[39m=[32m"video-player__container"[39m
      [33mdata-marko-key[39m=[32m"1 s0"[39m
      [33mstyle[39m=[32m"width:460px;height:300px"[39m
    [36m>[39m
      [36m<video[39m
        [33mno-controls[39m=[32m""[39m
      [36m>[39m
        [36m<source[39m
          [33msrc[39m=[32m"https://ir.ebaystatic.com/cr/v/c1/ebayui/video/v1/video.mp4"[39m
        [36m/>[39m
      [36m</video>[39m
    [36m</div>[39m
    [36m<div[39m
      [33mclass[39m=[32m"video-player__overlay video-player__overlay--hidden"[39m
    [36m>[39m
      [36m<svg[39m
        [33maria-hidden[39m=[32m"true"[39m
        [33mclass[39m=[32m"icon icon--24"[39m
        [33mfocusable[39m=[32m"false"[39m
      [36m>[39m
        [36m<defs>[39m
          [36m<symbol[39m
            [33mid[39m=[32m"icon-attention-24"[39m
            [33mviewBox[39m=[32m"0 0 24 24"[39m
          [36m>[39m
            [36m<path[39m
              [33md[39m=[32m"M12 0C5.373 0 0 5.373 0 12s5.373 12 12 12 12-5.373 12-12C23.994 5.375 18.625.006 12 0Zm0 22C6.477 22 2 17.523 2 12S6.477 2 12 2s10 4.477 10 10c-.006 5.52-4.48 9.994-10 10ZM11 7a1 1 0 1 1 2 0v6a1 1 0 1 1-2 0V7Zm1 11a1 1 0 1 0 0-2 1 1 0 0 0 0 2Z"[39m
            [36m/>[39m
          [36m</symbol>[39m
        [36m</defs>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-attention-24"[39m
        [36m/>[39m
      [36m</svg>[39m
      [36m<div[39m
        [33mclass[39m=[32m"video-player__overlay-text"[39m
      [36m>[39m
        [0mAn error has occurred[0m
      [36m</div>[39m
    [36m</div>[39m
    [36m<div[39m
      [33mclass[39m=[32m"video-player__overlay video-player__overlay--hidden"[39m
    [36m>[39m
      [36m<svg[39m
        [33maria-labelledby[39m=[32m"s0-9-0-text"[39m
        [33mclass[39m=[32m"icon icon--64-colored"[39m
        [33mfocusable[39m=[32m"false"[39m
        [33mrole[39m=[32m"img"[39m
      [36m>[39m
        [36m<defs>[39m
          [36m<symbol[39m
            [33mid[39m=[32m"icon-play-filled-64-colored"[39m
            [33mviewBox[39m=[32m"0 0 64 64"[39m
          [36m>[39m
            [36m<circle[39m
              [33mcx[39m=[32m"32"[39m
              [33mcy[39m=[32m"32"[39m
              [33mfill[39m=[32m"#fff"[39m
              [33mfill-opacity[39m=[32m".88"[39m
              [33mr[39m=[32m"32"[39m
            [36m/>[39m
            [36m<path[39m
              [33md[39m=[32m"m26.131 16.41 21.6 13.33a2.648 2.648 0 0 1 0 4.52l-21.6 13.33C24.333 48.7 22 47.425 22 45.33V18.67c0-2.094 2.333-3.37 4.131-2.26Z"[39m
              [33mfill[39m=[32m"#191919"[39m
            [36m/>[39m
          [36m</symbol>[39m
        [36m</defs>[39m
        [36m<title[39m
          [33mid[39m=[32m"s0-9-0-text"[39m
        [36m>[39m
          [0mClick to play[0m
        [36m</title>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-play-filled-64-colored"[39m
        [36m/>[39m
      [36m</svg>[39m
    [36m</div>[39m
    [36m<div[39m
      [33mstyle[39m=[32m"display:none"[39m
    [36m>[39m
      [36m<svg[39m
        [33maria-hidden[39m=[32m"true"[39m
        [33mclass[39m=[32m"icon icon--64-colored"[39m
        [33mfocusable[39m=[32m"false"[39m
      [36m>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-play-filled-64-colored"[39m
        [36m/>[39m
      [36m</svg>[39m
      [36m<svg[39m
        [33maria-hidden[39m=[32m"true"[39m
        [33mclass[39m=[32m"icon icon--24"[39m
        [33mfocusable[39m=[32m"false"[39m
      [36m>[39m
        [36m<defs>[39m
          [36m<symbol[39m
            [33mid[39m=[32m"icon-flag-24"[39m
            [33mviewBox[39m=[32m"0 0 24 24"[39m
          [36m>[39m
            [36m<path[39m
              [33mclip-rule[39m=[32m"evenodd"[39m
              [33md[39m=[32m"M20.777 1.37a.999.999 0 0 1-.207 1.452L14.72 7l5.832 4.166A.999.999 0 0 1 19.979 13H7v9a1 1 0 1 1-2 0V2a1 1 0 0 1 1-1h13.987a1.001 1.001 0 0 1 .79.37ZM16.88 11l-4.46-3.186a1 1 0 0 1 0-1.628L16.879 3H7v8h9.879Z"[39m
              [33mfill-rule[39m=[32m"evenodd"[39m
            [36m/>[39m
          [36m</symbol>[39m
        [36m</defs>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-flag-24"[39m
        [36m/>[39m
      [36m</svg>[39m
    [36m</div>[39m
  [36m</div>[39m
[36m</DocumentFragment>[39m"
`;

exports[`video > renders with report text 1`] = `
"[36m<DocumentFragment>[39m
  [36m<div[39m
    [33mclass[39m=[32m"video-player video-player--poster"[39m
  [36m>[39m
    [36m<div[39m
      [33mclass[39m=[32m"video-player__container"[39m
      [33mdata-marko-key[39m=[32m"1 s0"[39m
      [33mstyle[39m=[32m"width:600px;height:400px"[39m
    [36m>[39m
      [36m<video[39m
        [33mno-controls[39m=[32m""[39m
      [36m>[39m
        [36m<source[39m
          [33msrc[39m=[32m"https://ir.ebaystatic.com/cr/v/c1/ebayui/video/v1/playlist.mpd"[39m
          [33mtype[39m=[32m"dash"[39m
        [36m/>[39m
      [36m</video>[39m
    [36m</div>[39m
    [36m<div[39m
      [33mclass[39m=[32m"video-player__overlay video-player__overlay--hidden"[39m
    [36m>[39m
      [36m<svg[39m
        [33maria-hidden[39m=[32m"true"[39m
        [33mclass[39m=[32m"icon icon--24"[39m
        [33mfocusable[39m=[32m"false"[39m
      [36m>[39m
        [36m<defs>[39m
          [36m<symbol[39m
            [33mid[39m=[32m"icon-attention-24"[39m
            [33mviewBox[39m=[32m"0 0 24 24"[39m
          [36m>[39m
            [36m<path[39m
              [33md[39m=[32m"M12 0C5.373 0 0 5.373 0 12s5.373 12 12 12 12-5.373 12-12C23.994 5.375 18.625.006 12 0Zm0 22C6.477 22 2 17.523 2 12S6.477 2 12 2s10 4.477 10 10c-.006 5.52-4.48 9.994-10 10ZM11 7a1 1 0 1 1 2 0v6a1 1 0 1 1-2 0V7Zm1 11a1 1 0 1 0 0-2 1 1 0 0 0 0 2Z"[39m
            [36m/>[39m
          [36m</symbol>[39m
        [36m</defs>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-attention-24"[39m
        [36m/>[39m
      [36m</svg>[39m
      [36m<div[39m
        [33mclass[39m=[32m"video-player__overlay-text"[39m
      [36m>[39m
        [0mAn error has occurred[0m
      [36m</div>[39m
    [36m</div>[39m
    [36m<div[39m
      [33mclass[39m=[32m"video-player__overlay video-player__overlay--hidden"[39m
    [36m>[39m
      [36m<svg[39m
        [33maria-labelledby[39m=[32m"s0-9-0-text"[39m
        [33mclass[39m=[32m"icon icon--64-colored"[39m
        [33mfocusable[39m=[32m"false"[39m
        [33mrole[39m=[32m"img"[39m
      [36m>[39m
        [36m<defs>[39m
          [36m<symbol[39m
            [33mid[39m=[32m"icon-play-filled-64-colored"[39m
            [33mviewBox[39m=[32m"0 0 64 64"[39m
          [36m>[39m
            [36m<circle[39m
              [33mcx[39m=[32m"32"[39m
              [33mcy[39m=[32m"32"[39m
              [33mfill[39m=[32m"#fff"[39m
              [33mfill-opacity[39m=[32m".88"[39m
              [33mr[39m=[32m"32"[39m
            [36m/>[39m
            [36m<path[39m
              [33md[39m=[32m"m26.131 16.41 21.6 13.33a2.648 2.648 0 0 1 0 4.52l-21.6 13.33C24.333 48.7 22 47.425 22 45.33V18.67c0-2.094 2.333-3.37 4.131-2.26Z"[39m
              [33mfill[39m=[32m"#191919"[39m
            [36m/>[39m
          [36m</symbol>[39m
        [36m</defs>[39m
        [36m<title[39m
          [33mid[39m=[32m"s0-9-0-text"[39m
        [36m>[39m
          [0mClick to play[0m
        [36m</title>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-play-filled-64-colored"[39m
        [36m/>[39m
      [36m</svg>[39m
    [36m</div>[39m
    [36m<div[39m
      [33mstyle[39m=[32m"display:none"[39m
    [36m>[39m
      [36m<svg[39m
        [33maria-hidden[39m=[32m"true"[39m
        [33mclass[39m=[32m"icon icon--64-colored"[39m
        [33mfocusable[39m=[32m"false"[39m
      [36m>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-play-filled-64-colored"[39m
        [36m/>[39m
      [36m</svg>[39m
      [36m<svg[39m
        [33maria-hidden[39m=[32m"true"[39m
        [33mclass[39m=[32m"icon icon--24"[39m
        [33mfocusable[39m=[32m"false"[39m
      [36m>[39m
        [36m<defs>[39m
          [36m<symbol[39m
            [33mid[39m=[32m"icon-flag-24"[39m
            [33mviewBox[39m=[32m"0 0 24 24"[39m
          [36m>[39m
            [36m<path[39m
              [33mclip-rule[39m=[32m"evenodd"[39m
              [33md[39m=[32m"M20.777 1.37a.999.999 0 0 1-.207 1.452L14.72 7l5.832 4.166A.999.999 0 0 1 19.979 13H7v9a1 1 0 1 1-2 0V2a1 1 0 0 1 1-1h13.987a1.001 1.001 0 0 1 .79.37ZM16.88 11l-4.46-3.186a1 1 0 0 1 0-1.628L16.879 3H7v8h9.879Z"[39m
              [33mfill-rule[39m=[32m"evenodd"[39m
            [36m/>[39m
          [36m</symbol>[39m
        [36m</defs>[39m
        [36m<use[39m
          [33mhref[39m=[32m"#icon-flag-24"[39m
        [36m/>[39m
      [36m</svg>[39m
    [36m</div>[39m
  [36m</div>[39m
[36m</DocumentFragment>[39m"
`;
